app/workers/plum_event_handler.rb

```diff
@@ -6,6 +6,7 @@ class PlumEventHandler
              )
 
   def work(msg)
+    ActiveRecord::Base.connection.verify!
```

Comment:
What happens if this fails? The event stays on the queue for the next client?

Reply:
Yes